packages/lib/services/plugins/api/JoplinViewsPanels.test.ts

```
@@ -0,0 +1,17 @@
+/// <reference types="jest" />
```

ESLint

[error] Expected space or tab after '//' in comment.

(spaced-comment)